lib/SearchAndSort/components/DateRangeFilter/Field.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useField } from 'react-final-form';
+import { FormattedMessage } from 'react-intl';
+
+import { Datepicker } from '@folio/stripes/components';
+
+import { isDateValid } from './date-validations';
+
+export const Field = ({ name, dateFormat, ...rest }) => {
+  const validate = date => {
+    const dateIsEmpty = date === '';
+    const dateIsInvalid = !dateIsEmpty && !isDateValid(date, dateFormat);
+
+    if (dateIsInvalid) {
+      return (
+        <span data-test-invalid-date={name}>
+          <FormattedMessage id="stripes-smart-components.dateRange.invalidDate" />
+        </span>
+      );
+    }
+
+    return undefined;
```

Comment:
You can just leave `return;` it's returns `undefined` by default. On line 48 as well

Reply:
Sonar was not happy with that